base/src/types.rs

```diff
@@ -369,9 +369,11 @@ pub struct Font {
     #[serde(default = "default_as_false")]
     #[serde(skip_serializing_if = "is_false")]
     pub u: bool, // seems that Excel supports a bit more - double underline / account underline etc.
```

Comment:
Correct, you could open a ticket for this. And actually this might be a nice first issue.